README.md

```diff
@@ -124,10 +127,6 @@ A separate source folder [src/intTest](./src/intTest) contains integration tests
     
 ### Debugging
 
-#### MESH API
-
-A `mesh.sh` bash script exists for testing or debugging MESH. For more information see: [mesh/README.md](/mesh/README.md)
```

Comment:
Not part of this task, but on commit it was flagged the link does not exist

Reply:
I'll actually be putting this right back in again for NIAD-957, so perhaps keep it?

Reply:
Sure - i'll put this back then 👍

Reply:
TYVM!